packages/core/admin/admin/src/components/AuthenticatedApp.tsx

```diff
@@ -83,7 +83,9 @@ const AuthenticatedApp = () => {
       const isUserSuperAdmin = userRoles.find(({ code }) => code === 'strapi-super-admin');
 
       if (isUserSuperAdmin && appInfo?.autoReload) {
-        setGuidedTourVisibility(true);
+        setGuidedTourVisibility('super-admin');
+      } else if (!isUserSuperAdmin && appInfo?.autoReload) {
```

Comment:
Doesn't this check therefore mean we can change the whole block to just be `if appInfo.autoReload` and then change the user string we pass inline?